server/lib/validation/json-validator.test.ts

```diff
@@ -0,0 +1,34 @@
+import Joi from 'joi'
+import { json } from './json-validator'
+
+describe('server/lib/json/json-validator', () => {
+  test('throws an error if the property is not a valid JSON', () => {
```

Comment:
Can you add a test for when the invalid json is inside of a normal Joi object or something? To verify that the field label actually gets applied in the error message. Since the validation message here is using `value` for the root (I guess), it's hard to say whether that is working correctly